modules/ROOT/pages/sharepoint/sharepoint-connector.adoc

```diff
@@ -196,6 +196,18 @@ For Files and Folders, operations include add, retrieve content, retrieve metada
 * What Mule editions can I use this connector with?
 +
 This connector is supported on any Enterprise Edition Anypoint platform running on any operating system and bit type, including the CloudHub integration PaaS.
++
+* How do I build a query for a query operation?
++
+The query operations `Folder Query`, `File Query` and `List Item Query` only support OData queries.
++
+For the `List Item Query` operation the query should have this format: `LIST_ID?$select=FIELDS_TO_SELECT&filter=FILTER_PART`.
++
+For example: `Vessels?$select=Title,FLAG,SEGMENT,ID,VesselName&$filter=Title eq 1`.
++
+The same format also applies to the other query operations.
++
+You can learn more about building queries at the SharePoint documentation page https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/use-odata-query-operations-in-sharepoint-rest-requests[Use OData query operations in SharePoint REST requests].
```

Comment:
To learn more about building queries see the SharePoint documentation page https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/use-odata-query-operations-in-sharepoint-rest-requests[Use OData query operations in SharePoint REST requests].

Reply:
changed